### PR TITLE
Math magicians: components

### PR DESCRIPTION
**Pull request by hunter4466**
-These points were done in this project:

*Deleted all the boilerplate from CRA.
*Created a directory called components.
*Created a new Calculator.js file inside of components directory,
*Create a React component matching calculator design.
*Used class-based components.
*Imported the component in the main component (App.js).
*Tested it in the browser and rendered it without problems.
*Fixed Linters's problems.
*Added some styling on buttons hover and click.

### CODE REVIEW

File: src/components/calculator.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+// eslint-disable-next-line react/prefer-stateless-function
+export default class Calculator extends React.Component {
```
Comment (henatan99) [inline src/components/calculator.js]: - [ ] OPTIONAL:  `export default class Calculator extends Component` is preferred. Also make sure to change the import code to  `import React, { Component } from 'react'` if you want to take this OPTIONAL change into consideration.

File: src/index.js [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
-import './index.css';
-import App from './App';
+import './styles/_calculator.scss';
+import Calculator from './components/calculator';
 import reportWebVitals from './reportWebVitals';
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <Calculator />
```
Comment (henatan99) [inline src/index.js]: - [ ] You don't need to remove the `App` component even though you have only a single child component `Calculator`. For the sake of `React/JS best practice,` you should always use the `App` component as a wrapper. Kindly brig back `App.js` and render the `Calculator` component inside it. :pray:

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi @hunter4466 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._
- [ ] All the boilerplate that got created by the CRA (texts, images, and styles) ought to have gotten deleted. You have some of them in your public directory. Kindly remove them. :pray: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] OPTIONAL: Your commit messages are great, but it is recommended to write them in imperative sentence form. Just take this comment as advice for the way to go in the future. It is ok for this project. 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js [hunk 3]
```diff
@@ -1,26 +1,8 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/calculator';
 
 function App() {
   return (
```
Comment (Dipeshtwis) [inline src/App.js]: - [ ] The `App.js` component should be class-based as per the project's requirements even though the functional component is very correct. Please change this functional component to a class component. :+1:
- [ ] You also need to import `React, {Component}` when you convert the functional component to class component extended from `React Component`.

File: src/components/calculator.js [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+            <button type="button" className="left-btn">5</button>
+            <button type="button" className="left-btn">6</button>
+            <button type="button" className="left-btn">1</button>
+            <button type="button" className="left-btn">2</button>
+            <button type="button" className="left-btn">3</button>
+            <button type="button" className="left-btn-cero">0</button>
+            <button type="button" className="left-btn">.</button>
+          </div>
+          <div className="right-btns-container">
+            <button type="button" className="right-btn">+</button>
```
Comment (Dipeshtwis) [inline src/components/calculator.js]: - [ ] I think it should be `÷` (division sign) instead of `+` (addition) here.

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @hunter4466, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**

-------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mricanho — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @hunter4466  🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**